roles/tendrl-ansible.tendrl-server-cleanup/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# Task file for storage nodes
+
+- name: Stop tendrl services
+  service:
+    name: tendrl-node-agent
+    state: stopped
+
+- name: Stop etcd service
+  service:
+    name: etcd
+    state: stopped
+
+- name: Stop grafana-server service
+  service:
+    name: grafana-server
+    state: stopped
+
+- name: Stop carbon-cache service
+  service:
+    name: carbon-cache
+    state: stopped
+
+- name: Stop httpd service
+  service:
+    name: httpd
+    state: stopped
+
+- name: Cleanup etcd data
+  file:
+    path: "{{etcd_db_path}}/"
+    state: absent
+
+- name: Cleanup grafana data
+  file:
+    path: "{{grafana_db_file}}"
+    state: absent
+
+- name: Cleanup carbon data
```

Comment:
same as above

Reply:
In case of carbon we will delete only tendrl specific directory and nothing else.

Reply:
Correct. I missed that.